### ISSUE TITLE
Migration duplicate FK columns

### ISSUE THREAD

**Issue by KilroggD** [phalcon/phalcon-devtools#897]
Hello! I've got the same issue as https://github.com/phalcon/phalcon-devtools/issues/861 on MySQL
After migration generate command I can see something like this
```php
                    new Reference(
                        'images_author_fk',
                        [
                            'referencedSchema' => 'migrate',
                            'referencedTable' => 'users',
                            'columns' => ['author','author'],
                            'referencedColumns' => ['id','id'],
                            'onUpdate' => 'CASCADE',
                            'onDelete' => 'NO ACTION'
                        ]
                    )
```

And errors like  Column already exists: 1060 Duplicate column name 'image_id'

**Comment by sergeyklay** [comment]
@KilroggD Provide please Table schema and steps to reproduce

**Comment by KilroggD** [comment]
https://github.com/Arrim/phalcon-devtools/commit/353d07dee84a1835b38d8697cb0a5c4387a429c2 Actually, this fix helped me.

**Comment by sergeyklay** [comment]
@KilroggD Could you please send PR to the 3.0.x branch?

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
